samples/WorkerBindingSamples/NuGet.Config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="azure_app_service" value="https://www.myget.org/F/azure-appservice/api/v2" />
```

Comment:
make me a little nervous if this is in our sample; I don't want customers to be pulling prereleases packages from our  myget

Reply:
It worked without staging also